---
abstract: The haploid mammalian Y chromosome is usually under-represented in genome assemblies due to high repeat content and low depth due to its haploid nature. One strategy to ameliorate the low coverage of Y sequences is to experimentally enrich Y-specific material before assembly. As the enrichment process is imperfect, algorithms are needed to identify putative Y-specific reads prior to downstream assembly. A strategy that uses k-mer abundances to identify such reads was used to assemble the gorilla Y. However, the strategy required the manual setting of key parameters, a time-consuming process leading to sub-optimal assemblies. We develop a method, RecoverY, that selects Y-specific reads by automatically choosing the abundance level at which a k-mer is deemed to originate from the Y. This algorithm uses prior knowledge about the Y chromosome of a related species or known Y transcript sequences. We evaluate RecoverY on both simulated and real data, for human and gorilla, and investigate its robustness to important parameters. We show that RecoverY leads to a vastly superior assembly compared to alternate strategies of filtering the reads or contigs. Compared to the preliminary strategy used by Tomaszkiewicz et al., we achieve a 33% improvement in assembly size and a 20% improvement in the NG50, demonstrating the power of automatic parameter selection.
#author_notes:
#- Equal contribution
#- Equal contribution
authors:
- S Rangavittal
- RS Harris
- admin 
- M Tomaszkiewicz
- R Chikhi
- KD Makova
- P Medvedev
#- Rahulsimham Vegesna
date: "2017-11-28T00:00:00Z"
doi: ""
featured: false
image:
  caption: 'Image credit: DANI ZEMBA AND MONIKA CECHOVA, PENN STATE'
  focal_point: ""
  preview_only: false
projects: []
publication: '*Bioinformatics*'
publication_short: ""
publication_types:
- "2"
publishDate: "2017-11-28T00:00:00Z"
#slides: example
summary: The haploid mammalian Y chromosome is usually under-represented in genome assemblies due to high repeat content and low depth due to its haploid nature. One strategy to ameliorate the low coverage of Y sequences is to experimentally enrich Y-specific material before assembly. As the enrichment process is imperfect, algorithms are needed to identify putative Y-specific reads prior to downstream assembly. A strategy that uses k-mer abundances to identify such reads was used to assemble the gorilla Y. However, the strategy required the manual setting of key parameters, a time-consuming process leading to sub-optimal assemblies. We develop a method, RecoverY, that selects Y-specific reads by automatically choosing the abundance level at which a k-mer is deemed to originate from the Y. This algorithm uses prior knowledge about the Y chromosome of a related species or known Y transcript sequences. We evaluate RecoverY on both simulated and real data, for human and gorilla, and investigate its robustness to important parameters. We show that RecoverY leads to a vastly superior assembly compared to alternate strategies of filtering the reads or contigs. Compared to the preliminary strategy used by Tomaszkiewicz et al., we achieve a 33% improvement in assembly size and a 20% improvement in the NG50, demonstrating the power of automatic parameter selection.
tags:
- Source Themes
title: RecoverY -- k-mer-based read classification for Y-chromosome-specific sequencing and assembly
url_code: ""
url_dataset: ""
url_pdf: https://academic.oup.com/bioinformatics/article-pdf/34/7/1125/25119624/btx771.pdf
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---

#{{% alert note %}}
#Click the *Cite* button above to demo the feature to enable visitors to import #publication metadata into their reference management software.
#{{% /alert %}}

#{{% alert note %}}
#Click the *Slides* button above to demo Academic's Markdown slides feature.
#{{% /alert %}}

#Supplementary notes can be added here, including [code and #math](https://sourcethemes.com/academic/docs/writing-markdown-latex/).
